<ui version="4.0" >
 <class>NewWorkingTreeForm</class>
 <widget class="QWidget" name="NewWorkingTreeForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>428</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Branch</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Branch source (enter a URL or select a local directory with an exiting branch)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QComboBox" name="from_location" >
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy" >
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="from_picker" >
        <property name="text" >
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="label" >
        <property name="font" >
         <font>
          <underline>false</underline>
         </font>
        </property>
        <property name="text" >
         <string>Local directory where the working tree will be created</string>
        </property>
        <property name="openExternalLinks" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLineEdit" name="to_location" />
      </item>
      <item row="3" column="1" >
       <widget class="QPushButton" name="to_picker" >
        <property name="text" >
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Working Tree Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" colspan="2" >
       <widget class="QRadioButton" name="but_checkout" >
        <property name="text" >
         <string>Create a checkout</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="but_lightweight" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-style:italic;">Lightweight checkouts &lt;/span>depend on access to the branch for every operation.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Normal checkouts can perform common operations like diff and status without such access, and also support local commits.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Light-weight checkout</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QRadioButton" name="but_branch" >
        <property name="text" >
         <string>Make a local copy of the branch</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" >
       <widget class="QCheckBox" name="but_stacked" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="toolTip" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">A &lt;span style=" font-style:italic;">stacked branch&lt;/span> only stores information not in the source branch, and as such, depends on the availability of the source branch&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Create a stacked branch referring to the source branch</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2" >
       <widget class="QLabel" name="link_help" >
        <property name="font" >
         <font>
          <underline>true</underline>
         </font>
        </property>
        <property name="cursor" >
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; text-decoration: underline;">Checkouts&lt;/span>&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Checkouts are source trees that are connected to a branch, so that when&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">you commit in the source tree, the commit goes into that branch.  They&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">allow you to use a simpler, more centralized workflow, ignoring some of&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Bazaar's decentralized features until you want them. Using checkouts&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">with shared repositories is very similar to working with SVN or CVS, but&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">doesn't have the same restrictions.  And using checkouts still allows&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">others working on the project to use whatever workflow they like.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;">&lt;span style=" text-decoration: underline;">Branches&lt;/span>&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">A branch consists of the state of a project, including all of its&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">history. All branches have a repository associated (which is where the&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">branch history is stored), but multiple branches may share the same&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">repository (a shared repository). Branches can be copied and merged.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;a href="qbzrglossary:checkouts_and_branches">&lt;span style=" text-decoration: underline; color:#0000ff;">Tell me more about checkouts and branches.&lt;/span>&lt;/a>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="textFormat" >
         <enum>Qt::RichText</enum>
        </property>
        <property name="openExternalLinks" >
         <bool>false</bool>
        </property>
        <property name="textInteractionFlags" >
         <set>Qt::TextBrowserInteraction</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Revision</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" colspan="2" >
       <widget class="QRadioButton" name="but_rev_tip" >
        <property name="text" >
         <string>Most recent (tip) revision</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="but_rev_specific" >
        <property name="text" >
         <string>Revision:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="revision" />
      </item>
      <item row="1" column="2" >
       <spacer name="spacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>208</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3" >
       <widget class="QPushButton" name="but_show_log" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Show Log...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
